specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -320,6 +322,10 @@ function RecordSet<SCHEMA extends AnySchema>({
       )
     ).then(f.void);
 
+  const isTitlePresent = recordSet.get('name') === '' && !currentRecord.isNew();
```

Comment:
the variable is called `isTitlePresent`, but you are checking for `=== ''`? a bug, or poor variable naming?

Reply:
Wouldn't it be better to instead open the dialog when user tries to create a record set?

https://github.com/specify/specify7/blob/79fc28bcf32ea8326b1f8c613afeab3c8613e4b8/specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx#L336

(so instead of creating the record set, ask user for the name first)

and then, in the onSaved callback from ResourceView, call the createNewRecordSet function